doc/api/util.md

```diff
@@ -1399,6 +1399,35 @@ properties. Such objects are created either by Node.js internals or native
 addons. In JavaScript, they are [frozen][`Object.freeze()`] objects with a
 `null` prototype.
 
+```c++
+#include <js_native_api.h>
+#include <string.h>
```

Comment:
```suggestion
#include <stdlib.h>
```

(Unless I’m missing something, no `string.h` method is being used, only `malloc` from `stdlib.h`?)

Reply:
@addaleax , thanks. Fixed it.